src/resources/js/views/datepicker.js

```diff
@@ -65,6 +57,39 @@ tribe.events.views.datepicker = {};
 		initialized: false,
 	};
 
+	/**
+	 * Object of datepicker options
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.options = {
+		container: null,
+		daysOfWeekDisabled: [],
+		/**
+		 * @todo: use format from BE. Paul.
```

Comment:
Lets start using the `@` from github on these.

Reply:
copied these from other places, but will add.